config/resolver.go

```diff
@@ -56,6 +56,9 @@
 package config
```

Comment:
I would be cool to provide an example with the watch usage.

Reply:
done